packages/epics/src/agreements/components/agreement-head.tsx

```diff
@@ -25,20 +23,13 @@ export const AgreementHead = ({
 }: AgreementHeadProps) => {
   return (
     <div className="flex items-center">
-      <Skeleton
-        width="64px"
-        height="64px"
-        loading={isLoading}
-        className="rounded-lg mr-3"
-      >
-        <PersonAvatar
-          className="min-w-[64px] min-h-[64px] mr-3"
-          avatarSrc={creator?.avatar}
-          userName={`${creator?.name} ${creator?.surname}`}
-        />
-      </Skeleton>
-
-      <div className="flex justify-between items-center w-full">
+      <PersonAvatar
+        size="lg"
+        isLoading={isLoading}
+        avatarSrc={creator?.avatarUrl}
+        userName={`${creator?.name} ${creator?.surname}`}
+      />
+      <div className="flex justify-between items-center w-full ml-3">
```

Comment:
Now it's the same but different. [Check this commit](https://github.com/hypha-dao/hypha-web/pull/588/commits)